src/main/java/org/llorllale/cactoos/matchers/ScalarHasValue.java

```diff
@@ -38,6 +38,10 @@
  *
  * @param <T> Type of result
  * @since 0.2
+ * @todo #7:30min All the matchers should extend MatcherEnvelope and
```

Comment:
@Iprogrammerr here you need to reference the issue you tackled (#75) and not the original one (#7)